cmd/cost/cost.go

```diff
@@ -0,0 +1,95 @@
+package cost
+
+import (
+	"fmt"
+	"github.com/openshift/osd-utils-cli/cmd/common"
+	awsprovider "github.com/openshift/osd-utils-cli/pkg/provider/aws"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"log"
```

Comment:
It would be good to group and sort all the imports. But this is not urgent, we can definitely do it in follow-up PRs and add linters to ensure all our imports are sorted.

Reply:
I think `gofmt` should sort these